x-pack/plugins/rule_registry/server/routes/get_alert_summary.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import Boom from '@hapi/boom';
+import { IRouter } from '@kbn/core/server';
+import * as t from 'io-ts';
+import { transformError } from '@kbn/securitysolution-es-utils';
+import moment from 'moment';
+
+import { RacRequestHandlerContext } from '../types';
+import { BASE_RAC_ALERTS_API_PATH } from '../../common/constants';
+import { buildRouteValidation } from './utils/route_validation';
+
+export const getAlertSummaryRoute = (router: IRouter<RacRequestHandlerContext>) => {
+  router.post(
+    {
+      path: `${BASE_RAC_ALERTS_API_PATH}/_alert_summary`,
+      validate: {
+        body: buildRouteValidation(
+          t.intersection([
+            t.exact(
+              t.type({
+                gte: t.string,
+                lte: t.string,
+              })
+            ),
+            t.exact(
+              t.partial({
+                featureIds: t.array(t.string),
```

Comment:
does this mean the endpoint will only accept an array of strings? If so maybe we can remove the type checking between `string` and `string[]` for `featureIds` in `getAlertSummary`